api/webapp/server/handler/middleware/gitoauth2/getUserGit.go

```diff
@@ -0,0 +1,51 @@
+package gitoauth2
+
+import (
+	"encoding/json"
+	"fmt"
+	"goReact/domain/store"
+	"goReact/webapp/server/handler"
+	"goReact/webapp/server/handler/response"
+	"net/http"
+	"os"
+
+	"golang.org/x/oauth2"
+)
+
+var getUserURI = os.Getenv("GIT_GET_USER_DATA") 
+
+func GetUserGit(s *store.Store) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		token := r.Context().Value(handler.CTXKeyAccessTokenGitOAuth).(*oauth2.Token)
+
+		authHeader := fmt.Sprintf("token %s", token.AccessToken)
+
+		w.Header().Set("Accept", "application/json")		
+		
+		req, err := http.NewRequest(http.MethodGet, getUserURI, nil)
```

Comment:
I would say that this should go inside service that would be responsible for working with git SSO. Because otherwise this middleware has multiple responsibilities

Reply:
done